db/schema.rb

```diff
@@ -10,76 +10,75 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_10_30_174955) do
-
+ActiveRecord::Schema.define(version: 2023_10_28_225001) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_enum :adjustment_kind, [
     "fine",
     "membership",
     "donation",
-    "payment",
+    "payment"
```

Comment:
The churn in this file I think stems from standard not getting applied to `schema.rb`, or possibly config change between versions. When I run standard against the `schema.rb` in main, I see these changes there as well.